pkg/llms/llms.go

```diff
@@ -29,10 +29,14 @@ const (
 	OpenAI    LLMType = "openai"
 	ZhiPuAI   LLMType = "zhipuai"
 	DashScope LLMType = "dashscope"
+	Gemini    LLMType = "gemini"
 	Unknown   LLMType = "unknown"
 )
 
-var OpenAIModels = []string{"gpt-3.5", "gpt-3.5-turbo"}
+var (
+	OpenAIModels = []string{"gpt-3.5", "gpt-3.5-turbo"}
+	GeminiModels = []string{"gemini-pro"}
```

Comment:
Should be 
- gemini-1.0-pro
-  gemini-1.0-pro-vision
See https://cloud.google.com/vertex-ai/docs/generative-ai/learn/models#gemini-models

Besides  google's generative-ai-go has changed the name too https://github.com/google/generative-ai-go/commit/223b9f68d6fd478fd79b395478d7ea799f4b9d94

Reply:
For `gemini-1.0-pro-vision` ,I am  not sure if this is allowed.

Reply:
models/gemini-1.0-pro The best model for scaling across a wide range of tasks
models/gemini-1.0-pro-001 The best model for scaling across a wide range of tasks. This is a stable model that supports tuning.
models/gemini-1.0-pro-latest The best model for scaling across a wide range of tasks. This is the latest model.
models/gemini-1.0-pro-vision-latest The best image understanding model to handle a broad range of applications
models/gemini-pro The best model for scaling across a wide range of tasks
models/gemini-pro-vision The best image understanding model to handle a broad range of applications
models/embedding-001 Obtain a distributed representation of a text.

by https://pkg.go.dev/github.com/google/generative-ai-go/genai#Client.ListModels

Reply:
And "Note: gemini-pro is an alias for gemini-1.0-pro." in https://ai.google.dev/models/gemini